src/SFA.DAS.EmployerCommitmentsV2.Web/Models/Apprentice/SendNewTrainingProviderViewModel.cs

```diff
@@ -0,0 +1,20 @@
+﻿using SFA.DAS.Authorization.ModelBinding;
+using SFA.DAS.CommitmentsV2.Types;
+
+namespace SFA.DAS.EmployerCommitmentsV2.Web.Models.Apprentice
+{
+    public class SendNewTrainingProviderViewModel : IAuthorizationContextModel
+    {
+        public string AccountHashedId { get; set; }
+        public long AccountId { get; set; }
```

Comment:
AccountId and ApprenticeshipId are never used, can they be removed?

Reply:
AccountId & ApprenticeshipId will be required when we create the Cohort